### PR TITLE
Avatar and badges components 

### PR DESCRIPTION
**Pull request by saketkum11**
Please review .

Avatar
image

Badge 

icon badge
image

icon- avatar.
image

### CODE REVIEW

File: component/badges/badge.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.icon{
+    border: 0.125rem solid black;
```
Comment (t007rushi) [inline component/badges/badge.css]: Use px here

File: component/badges/badge.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+.icon-badge {
+    border: none;
+    outline: none;
+    width: 3rem;
+    height: 2.5rem;
+}
+.badge {
+    background-color: #F44336;
```
Comment (t007rushi) [inline component/badges/badge.css]: Use css variable for colors

File: component/badges/badge.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    width: 20px;
+    height: 20px;
```
Comment (t007rushi) [inline component/badges/badge.css]: Use rem here for responsive approach

File: utilities-css/universal.css [hunk 4]
```diff
@@ -1,22 +1,309 @@
...
+.lg {
+    width: 4rem;
+    height: 4rem;
+}
+
+.flex {
+    display: flex;
+}
+
+.flex-row {
```
Comment (t007rushi) [inline utilities-css/universal.css:238]: I think it's default so need of it

**Review by t007rushi — CHANGES_REQUESTED** [review]
Make suggested changes

File: component/badges/badge.css (hunk 1 above)
Comment (saketkum11) [reply component/badges/badge.css]: okk

File: component/badges/badge.css (hunk 3 above)
Comment (saketkum11) [reply component/badges/badge.css]: sure will add

File: component/badges/badge.css (hunk 2 above)
Comment (saketkum11) [reply component/badges/badge.css]: sure

File: utilities-css/universal.css (hunk 4 above)
Comment (saketkum11) [reply utilities-css/universal.css:238]: Its in utility css

**Review by Pratik1005 — APPROVED** [review]

**Review by t007rushi — APPROVED** [review]
Good to go

**Comment by sharathnair9999** [comment]
Good to merge for me

**Pull request merged**